charts/client/chart.scss

```diff
@@ -182,6 +182,7 @@ $zindex-IndicatorDropdown: 50;
     --------------------------------------------- */
 
     $footerRowHeight: 40px;
+    $footerRowHeightMobile: 30px;
```

Comment:
Just wondering why we shrink the footer height on mobile? Is this related or unrelated to adding download text?

Reply:
Requested by Ernst to increase space on mobile. I believe related to adding the download text.
https://www.notion.so/owid/Users-can-t-find-DATA-download-in-Chart-9df5fea8564b43c996f2b82a07f73936